src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -301,6 +341,89 @@ private boolean indexExists(final File vcf) {
                 unClassifiedStatesMap.put(condition, count);
             }
 
+            // write out an annotated VCF
+            if (writer != null) {
+                final VariantContextBuilder builder;
+                VariantContext truthContext = null, callContext = null;
+                final List<Genotype> genotypes = new ArrayList<>(2);
+
+                // get the variant contexts and initialize the output variant context builder
+                if (tuple.leftVariantContext.isPresent()) {
+                    truthContext = tuple.leftVariantContext.get();
+                }
+                if (tuple.rightVariantContext.isPresent()) {
+                    callContext = tuple.rightVariantContext.get();
+                }
+
+                // Get the alleles for each genotype
+                final Alleles alleles = normalizeAlleles(truthContext, TRUTH_SAMPLE, callContext, CALL_SAMPLE);
+
+                // Ignore if both are one of mixed, filtered, or missing.
```

Comment:
Is this comment still accurate?  I.e. alleles will be empty if both genotypes are mixed/filtered/missing?  If so, maybe reword to make that clear?